conf/application.conf

```diff
@@ -136,6 +136,9 @@ tracking-consent-frontend {
 features {
   welsh-translation = false
   enrolmentStoreProxyStubEnabled = true
+  privateBeta {
+      enabled = true
+    }
```

Comment:
very minor but formatting is slightly out here

Reply:
Ah yes 😄 . Fixed now.